bubble_sort.rb

```diff
@@ -0,0 +1,26 @@
+def sorting(array)
+  array.length.times do |arr_1|
+    (array.length - (arr_1 + 1)).times do |arr|
+      if array[arr + 1] != nil
```

Comment:
The ruby way to check for nil values is `.nil?` method. Use `if !array[arr + 1].nil?` instead